src/main/resources/assets/natura/lang/de_DE.lang

```diff
@@ -0,0 +1,582 @@
+##########################
+#    Overworld Blocks    #
+##########################
+
+tile.natura.clouds.white.name=Wolke
+tile.natura.clouds.white.tooltip=So leicht und flauschig!\nRettet Dich, wenn Du fällst
+tile.natura.clouds.dark.name=Dunkle Wolke
+tile.natura.clouds.dark.tooltip=So leicht und flauschig!\nVerursacht Blitzeinschläge
+tile.natura.clouds.ash.name=Aschewolke
+tile.natura.clouds.ash.tooltip=So leicht und flauschig!\n3x3 Verwandelt sich in Holzkohle
+tile.natura.clouds.sulfur.name=Schwefelwolke
+tile.natura.clouds.sulfur.tooltip=So leicht und flauschig!\n2x2 Verwandelt sich in Schwefel
+
+tile.natura.overworld_logs.maple.name=Ahornholz
+tile.natura.overworld_logs.maple.tooltip=Etwas süßlich
+tile.natura.overworld_logs.silverbell.name=Schneeglöckchenbaumholz
+tile.natura.overworld_logs.silverbell.tooltip=Schneeglöckchen
+tile.natura.overworld_logs.amaranth.name=Amaranthholz
+tile.natura.overworld_logs.amaranth.tooltip=Herz aus Holz
+tile.natura.overworld_logs.tiger.name=Tigerholz
+tile.natura.overworld_logs.tiger.tooltip=Wildgetreide
+
+tile.natura.overworld_logs2.willow.name=Weidenholz
+tile.natura.overworld_logs2.willow.tooltip=Der Weinende
+tile.natura.overworld_logs2.eucalyptus.name=Eukalyptusholz
+tile.natura.overworld_logs2.eucalyptus.tooltip=Das rosa Holz
+tile.natura.overworld_logs2.sakura.name=Blütenkirschholz
+tile.natura.overworld_logs2.sakura.tooltip=Blühende Kirsche
```

Comment:
On that note, this one is a bit redundant since it's already in the name of the plant, maybe change this too. One idea I had is just "Herz­er­wär­mend" since it's pretty to look at.

Reply:
Yes, that's good, I, again, just read the Wikipedia article to find an idea and found out that in Japan, the sakura stands for beauty, departure and transience.
So I thought we could go "Schön, aber vergänglich", but it isn't transient in Minecraft, so that doesn't really fit. We may say "Schön und unvergänglich" to draw attention to that issue.
We could also make it even more confusing and go for "Schöner Aufbruch". But I don't know if that's wanted.

Reply:
I like "Schön und unvergänglich".

Reply:
OK, I'll create a commit to fix both issues. :)